stand/i386/boot2/Makefile

```diff
@@ -63,10 +63,13 @@ CLEANFILES+=	boot2 boot2.ld boot2.ldr boot2.bin boot2.out boot2.o \
 
 BOOT2SIZE=	7680
 
+# Seek to BOOT2SIZE to ensure zero padding at the end
 boot2: boot2.ld
 	@set -- `ls -l ${.ALLSRC}`; x=$$((${BOOT2SIZE}-$$5)); \
 	    echo "$$x bytes available"; test $$x -ge 0
-	${DD} if=${.ALLSRC} of=${.TARGET} obs=${BOOT2SIZE} conv=osync
+	${DD} if=${.ALLSRC} of=${.TARGET}
```

Comment:
Seems simpler to just do

```
${DD} if=${.ALLSRC} of=${.TARGET} bs=${BOOT2SIZE} conv=sync
```

This works on FreeBSD, OS X, and Linux for me.  It works by padding the input with zeros to the block size before it is copied to the output.  You could use this for all of the cases you patched here just s/obs/bs/ and s/osync/sync/.

Reply:
Sounds good, I'll do that since it's simpler.